lib/mapper.js

```diff
@@ -24,7 +24,17 @@ exports.identify = function(identify){
   var userId = identify.userId();
   var anonymousId = identify.sessionId();
   var payload = {};
+  var ip = identify.ip();
+  var shouldTrackIp = identify.proxy('context.Kissmetrics.trackIp');
```

Comment:
Ah, I assumed the comment only applied to the first line (`context.ignoreIp`), and I assumed that the second line (`context.Mixpanel.ignoreIp`) was the preferred method of implementation.